test/scenario1.js

```diff
@@ -48,7 +48,7 @@ describe('Scenario 1 - Creating safe offChain', async () => {
   })
   //Step 1: Register all users
   it('Should register a Creator', async () => {
-    creatorSc = new SafientCore(creatorSigner, Enums.NetworkType.LOCALHOST, 'threadDB', apiKey, secret, null);
```

Comment:
You haven't updated the case in the enum file. Tests are failing.

Reply:
@koshikraj this is outdated code. The new code has updated enum keys. Look into it once.